rerun failed tests

osmaczko opened the pull request:
Review commit by commit.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (151)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | a951a05e | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/1/) | 2024-02-22 15:53:05 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240222-154943-a951a0-pr4796.zip) |
| :heavy_check_mark: | a951a05e | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/1/) | 2024-02-22 15:54:56 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240222-154943-a951a0-pr4796.zip) |
| :heavy_check_mark: | a951a05e | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/1/) | 2024-02-22 15:55:05 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240222-154943-a951a0-pr4796.aar) |
| :heavy_check_mark: | a951a05e | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/1/) | 2024-02-22 16:26:30 | ~36 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 92a18766 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/2/) | 2024-02-23 10:24:30 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-102107-92a187-pr4796.zip) |
| :heavy_check_mark: | 92a18766 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/2/) | 2024-02-23 10:24:52 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-102106-92a187-pr4796.zip) |
| :heavy_check_mark: | 92a18766 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/2/) | 2024-02-23 10:27:10 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-102106-92a187-pr4796.aar) |
| :heavy_multiplication_x: | 92a18766 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/2/) | 2024-02-23 12:55:17 | ~2 hr 34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/2//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | e5d32885 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/3/) | 2024-02-23 14:29:51 | ~46 sec | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/3//consoleText) |
| :heavy_check_mark: | e5d32885 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/3/) | 2024-02-23 14:30:34 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-142900-e5d328-pr4796.zip) |
| :heavy_check_mark: | e5d32885 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/3/) | 2024-02-23 14:30:42 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-142900-e5d328-pr4796.aar) |
| :heavy_check_mark: | e5d32885 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/3/) | 2024-02-23 14:32:26 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-142900-e5d328-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 00917e63 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/4/) | 2024-02-23 14:39:44 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-143635-00917e-pr4796.zip) |
| :heavy_check_mark: | 00917e63 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/4/) | 2024-02-23 14:39:47 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-143635-00917e-pr4796.zip) |
| :heavy_multiplication_x: | 00917e63 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/4/) | 2024-02-23 14:39:52 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/4//consoleText) |
| :heavy_check_mark: | 00917e63 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/4/) | 2024-02-23 14:42:24 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-143635-00917e-pr4796.aar) |
| | | | | | | |
| :heavy_check_mark: | 13a4786f | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/5/) | 2024-02-23 15:09:32 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-150820-13a478-pr4796.zip) |
| :heavy_multiplication_x: | 13a4786f | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/5/) | 2024-02-23 15:09:33 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/5//consoleText) |
| :heavy_check_mark: | 13a4786f | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/5/) | 2024-02-23 15:10:03 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-150820-13a478-pr4796.aar) |
| :heavy_check_mark: | 13a4786f | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/5/) | 2024-02-23 15:11:47 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-150820-13a478-pr4796.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 31af2b7a | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/6/) | 2024-02-23 17:51:49 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/6//consoleText) |
| :heavy_check_mark: | 31af2b7a | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/6/) | 2024-02-23 17:52:19 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-175032-31af2b-pr4796.aar) |
| :heavy_check_mark: | 31af2b7a | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/6/) | 2024-02-23 17:53:45 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-175036-31af2b-pr4796.zip) |
| :heavy_check_mark: | 31af2b7a | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/6/) | 2024-02-23 17:54:07 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-175032-31af2b-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 3c5242ed | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/7/) | 2024-02-23 17:58:42 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-175712-3c5242-pr4796.zip) |
| :heavy_check_mark: | 3c5242ed | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/7/) | 2024-02-23 17:58:59 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-175712-3c5242-pr4796.aar) |
| :heavy_multiplication_x: | 3c5242ed | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/7/) | 2024-02-23 18:00:03 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/7//consoleText) |
| :heavy_check_mark: | 3c5242ed | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/7/) | 2024-02-23 18:00:47 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-175712-3c5242-pr4796.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 35e2849e | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/8/) | 2024-02-23 18:03:22 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/8//consoleText) |
| :heavy_check_mark: | 35e2849e | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/8/) | 2024-02-23 18:03:36 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-180213-35e284-pr4796.zip) |
| :heavy_check_mark: | 35e2849e | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/8/) | 2024-02-23 18:03:47 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-180210-35e284-pr4796.aar) |
| :heavy_check_mark: | 35e2849e | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/8/) | 2024-02-23 18:06:02 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-180212-35e284-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 20942aba | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/9/) | 2024-02-23 18:06:22 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-180457-20942a-pr4796.zip) |
| :heavy_check_mark: | 20942aba | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/9/) | 2024-02-23 18:06:48 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-180456-20942a-pr4796.aar) |
| :heavy_check_mark: | 20942aba | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/9/) | 2024-02-23 18:09:29 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-180605-20942a-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | db298e72 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/10/) | 2024-02-23 21:50:51 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-214918-db298e-pr4796.zip) |
| :heavy_check_mark: | db298e72 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/10/) | 2024-02-23 21:51:04 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/10//consoleText) |
| :heavy_check_mark: | db298e72 | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/10/) | 2024-02-23 21:51:31 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-214914-db298e-pr4796.aar) |
| :heavy_check_mark: | db298e72 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/10/) | 2024-02-23 21:52:54 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-214918-db298e-pr4796.zip) |
| | | | | | | |
| :heavy_multiplication_x: | df4a315b | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/11/) | 2024-02-23 21:57:51 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/11//consoleText) |
| :heavy_check_mark: | df4a315b | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/11/) | 2024-02-23 21:58:00 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-215638-df4a31-pr4796.zip) |
| :heavy_check_mark: | df4a315b | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/11/) | 2024-02-23 21:58:11 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-215634-df4a31-pr4796.aar) |
| :heavy_check_mark: | df4a315b | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/11/) | 2024-02-23 22:00:17 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-215638-df4a31-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | d621af85 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/12/) | 2024-02-23 22:04:25 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-220254-d621af-pr4796.zip) |
| :heavy_multiplication_x: | d621af85 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/12/) | 2024-02-23 22:04:47 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/12//consoleText) |
| :heavy_check_mark: | d621af85 | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/12/) | 2024-02-23 22:05:09 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-220254-d621af-pr4796.aar) |
| :heavy_check_mark: | d621af85 | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/12/) | 2024-02-23 22:06:31 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-220254-d621af-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 80c52ab2 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/13/) | 2024-02-23 22:11:30 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/13//consoleText) |
| :heavy_check_mark: | 80c52ab2 | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/13/) | 2024-02-23 22:11:39 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-221019-80c52a-pr4796.zip) |
| :heavy_check_mark: | 80c52ab2 | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/13/) | 2024-02-23 22:11:55 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-221019-80c52a-pr4796.aar) |
| :heavy_check_mark: | 80c52ab2 | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/13/) | 2024-02-23 22:13:20 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-221019-80c52a-pr4796.zip) |
| :heavy_check_mark: | 80c52ab2 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/14/) | 2024-02-23 22:13:47 | ~20 sec | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/14//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 946e2be1 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/15/) | 2024-02-23 22:17:02 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/15//consoleText) |
| :heavy_check_mark: | 946e2be1 | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/14/) | 2024-02-23 22:17:08 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-221553-946e2b-pr4796.zip) |
| :heavy_check_mark: | 946e2be1 | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/14/) | 2024-02-23 22:17:22 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-221549-946e2b-pr4796.aar) |
| :heavy_check_mark: | 946e2be1 | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/14/) | 2024-02-23 22:18:53 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-221553-946e2b-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | a72335ea | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/15/) | 2024-02-23 22:22:38 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240223-222113-a72335-pr4796.zip) |
| :heavy_check_mark: | a72335ea | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/15/) | 2024-02-23 22:22:43 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240223-222109-a72335-pr4796.aar) |
| :heavy_check_mark: | a72335ea | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/15/) | 2024-02-23 22:24:24 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240223-222113-a72335-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 141e6562 | [#16](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/16/) | 2024-02-24 07:10:37 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240224-070859-141e65-pr4796.zip) |
| :heavy_check_mark: | 141e6562 | [#16](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/16/) | 2024-02-24 07:10:48 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240224-070855-141e65-pr4796.aar) |
| :heavy_check_mark: | 141e6562 | [#16](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/16/) | 2024-02-24 07:12:24 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240224-070859-141e65-pr4796.zip) |
| :heavy_multiplication_x: | 141e6562 | [#17](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/17/) | 2024-02-24 09:38:40 | ~2 hr 29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/17//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 710c855f | [#17](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/17/) | 2024-02-24 16:44:04 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240224-164239-710c85-pr4796.zip) |
| :heavy_check_mark: | 710c855f | [#17](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/17/) | 2024-02-24 16:44:21 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240224-164235-710c85-pr4796.aar) |
| :heavy_check_mark: | 710c855f | [#17](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/17/) | 2024-02-24 16:46:02 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240224-164239-710c85-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 3e24d07a | [#18](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/18/) | 2024-02-24 16:49:05 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240224-164746-3e24d0-pr4796.zip) |
| :heavy_check_mark: | 3e24d07a | [#18](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/18/) | 2024-02-24 16:49:17 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240224-164742-3e24d0-pr4796.aar) |
| :heavy_check_mark: | 3e24d07a | [#18](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/18/) | 2024-02-24 16:50:53 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240224-164744-3e24d0-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | cf43c5b2 | [#19](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/19/) | 2024-02-24 16:54:42 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240224-165321-cf43c5-pr4796.zip) |
| :heavy_check_mark: | cf43c5b2 | [#19](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/19/) | 2024-02-24 16:55:20 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240224-165321-cf43c5-pr4796.aar) |
| :heavy_check_mark: | cf43c5b2 | [#19](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/19/) | 2024-02-24 16:57:10 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240224-165321-cf43c5-pr4796.zip) |
| :heavy_multiplication_x: | cf43c5b2 | [#20](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/20/) | 2024-02-24 18:31:42 | ~1 hr 38 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/20//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 8220119b | [#20](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/20/) | 2024-02-25 09:14:37 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240225-091315-822011-pr4796.zip) |
| :heavy_check_mark: | 8220119b | [#20](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/20/) | 2024-02-25 09:15:09 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240225-091310-822011-pr4796.aar) |
| :heavy_check_mark: | 8220119b | [#20](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/20/) | 2024-02-25 09:16:39 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240225-091315-822011-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | eed0787a | [#21](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/21/) | 2024-02-25 09:17:30 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240225-091600-eed078-pr4796.zip) |
| :heavy_check_mark: | eed0787a | [#21](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/21/) | 2024-02-25 09:17:42 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240225-091600-eed078-pr4796.aar) |
| :heavy_check_mark: | eed0787a | [#21](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/21/) | 2024-02-25 09:19:51 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240225-091642-eed078-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 2afe7b94 | [#22](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/22/) | 2024-02-25 14:37:17 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240225-143539-2afe7b-pr4796.zip) |
| :heavy_check_mark: | 2afe7b94 | [#22](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/22/) | 2024-02-25 14:37:20 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240225-143539-2afe7b-pr4796.aar) |
| :heavy_check_mark: | 2afe7b94 | [#22](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/22/) | 2024-02-25 14:38:45 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240225-143539-2afe7b-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 37ed958e | [#23](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/23/) | 2024-02-25 14:38:39 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240225-143720-37ed95-pr4796.zip) |
| :heavy_check_mark: | 37ed958e | [#23](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/23/) | 2024-02-25 14:38:58 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240225-143723-37ed95-pr4796.aar) |
| :heavy_check_mark: | 37ed958e | [#23](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/23/) | 2024-02-25 14:41:33 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240225-143848-37ed95-pr4796.zip) |
| :heavy_multiplication_x: | 37ed958e | [#24](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/24/) | 2024-02-25 15:06:53 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/24//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 7c7461f6 | [#24](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/24/) | 2024-02-25 16:39:54 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240225-163810-7c7461-pr4796.zip) |
| :heavy_check_mark: | 7c7461f6 | [#24](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/24/) | 2024-02-25 16:40:24 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240225-163810-7c7461-pr4796.aar) |
| :heavy_check_mark: | 7c7461f6 | [#24](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/24/) | 2024-02-25 16:41:46 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240225-163810-7c7461-pr4796.zip) |
| :heavy_multiplication_x: | 7c7461f6 | [#25](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/25/) | 2024-02-25 17:44:03 | ~1 hr 5 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/25//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 6c3c4670 | [#25](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/25/) | 2024-02-25 21:06:14 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240225-210458-6c3c46-pr4796.zip) |
| :heavy_check_mark: | 6c3c4670 | [#25](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/25/) | 2024-02-25 21:07:22 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240225-210458-6c3c46-pr4796.aar) |
| :heavy_check_mark: | 6c3c4670 | [#25](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/25/) | 2024-02-25 21:08:19 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240225-210458-6c3c46-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 11f2c7b1 | [#26](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/26/) | 2024-02-25 21:08:12 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240225-210648-11f2c7-pr4796.zip) |
| :heavy_check_mark: | 11f2c7b1 | [#26](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/26/) | 2024-02-25 21:09:03 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240225-210725-11f2c7-pr4796.aar) |
| :heavy_check_mark: | 11f2c7b1 | [#26](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/26/) | 2024-02-25 21:11:32 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240225-210823-11f2c7-pr4796.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 0972be8d | [#29](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/29/) | 2024-02-26 09:18:10 | ~44 sec | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/29//consoleText) |
| :heavy_multiplication_x: | 0972be8d | [#30](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/30/) | 2024-02-26 09:19:30 | ~39 sec | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/30//consoleText) |
| :heavy_check_mark: | 0972be8d | [#28](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/28/) | 2024-02-26 09:19:51 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-091824-0972be-pr4796.zip) |
| :heavy_check_mark: | 0972be8d | [#28](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/28/) | 2024-02-26 09:20:41 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-091849-0972be-pr4796.aar) |
| :heavy_check_mark: | 0972be8d | [#28](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/28/) | 2024-02-26 09:24:04 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-092038-0972be-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | e4b767be | [#27](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/27/) | 2024-02-26 09:18:21 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-091629-e4b767-pr4796.zip) |
| :heavy_check_mark: | e4b767be | [#27](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/27/) | 2024-02-26 09:18:42 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-091625-e4b767-pr4796.aar) |
| :heavy_check_mark: | e4b767be | [#27](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/27/) | 2024-02-26 09:20:35 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-091629-e4b767-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 8e496086 | [#29](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/29/) | 2024-02-26 09:24:03 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-092225-8e4960-pr4796.zip) |
| :heavy_check_mark: | 8e496086 | [#29](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/29/) | 2024-02-26 09:24:34 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-092225-8e4960-pr4796.aar) |
| :heavy_check_mark: | 8e496086 | [#29](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/29/) | 2024-02-26 09:27:22 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-092408-8e4960-pr4796.zip) |
| :heavy_multiplication_x: | 8e496086 | [#31](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/31/) | 2024-02-26 09:55:56 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/31//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 9c9b3dfb | [#30](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/30/) | 2024-02-26 10:29:05 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-102731-9c9b3d-pr4796.zip) |
| :heavy_check_mark: | 9c9b3dfb | [#30](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/30/) | 2024-02-26 10:29:23 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-102731-9c9b3d-pr4796.aar) |
| :heavy_check_mark: | 9c9b3dfb | [#30](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/30/) | 2024-02-26 10:32:32 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-102732-9c9b3d-pr4796.zip) |
| :heavy_multiplication_x: | 9c9b3dfb | [#32](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/32/) | 2024-02-26 11:02:17 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/32//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 5152dfa2 | [#31](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/31/) | 2024-02-26 13:28:17 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-132642-5152df-pr4796.zip) |
| :heavy_check_mark: | 5152dfa2 | [#31](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/31/) | 2024-02-26 13:28:24 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-132636-5152df-pr4796.aar) |
| :heavy_check_mark: | 5152dfa2 | [#31](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/31/) | 2024-02-26 13:29:58 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-132639-5152df-pr4796.zip) |
| :heavy_multiplication_x: | 5152dfa2 | [#33](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/33/) | 2024-02-26 14:01:33 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/33//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 15b6a30c | [#32](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/32/) | 2024-02-26 14:20:56 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-141939-15b6a3-pr4796.zip) |
| :heavy_check_mark: | 15b6a30c | [#32](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/32/) | 2024-02-26 14:21:19 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-141937-15b6a3-pr4796.aar) |
| :heavy_check_mark: | 15b6a30c | [#32](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/32/) | 2024-02-26 14:22:44 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-141938-15b6a3-pr4796.zip) |
| | | | | | | |
| :heavy_multiplication_x: | c43ba300 | [#35](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/35/) | 2024-02-26 14:22:37 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/35//consoleText) |
| :heavy_check_mark: | c43ba300 | [#33](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/33/) | 2024-02-26 14:22:45 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-142123-c43ba3-pr4796.zip) |
| :heavy_check_mark: | c43ba300 | [#33](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/33/) | 2024-02-26 14:23:13 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-142123-c43ba3-pr4796.aar) |
| :heavy_check_mark: | c43ba300 | [#33](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/33/) | 2024-02-26 14:25:58 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-142248-c43ba3-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 0ca391d4 | [#34](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/34/) | 2024-02-26 15:06:50 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-150540-0ca391-pr4796.zip) |
| :heavy_check_mark: | 0ca391d4 | [#34](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/34/) | 2024-02-26 15:07:21 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-150540-0ca391-pr4796.aar) |
| :heavy_check_mark: | 0ca391d4 | [#34](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/34/) | 2024-02-26 15:08:45 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-150540-0ca391-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 605304ae | [#35](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/35/) | 2024-02-26 15:09:17 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-150800-605304-pr4796.zip) |
| :heavy_check_mark: | 605304ae | [#35](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/35/) | 2024-02-26 15:09:29 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-150800-605304-pr4796.aar) |
| :heavy_check_mark: | 605304ae | [#35](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/35/) | 2024-02-26 15:11:38 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-150848-605304-pr4796.zip) |
| :heavy_multiplication_x: | 605304ae | [#37](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/37/) | 2024-02-26 19:56:28 | ~4 hr 48 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/37//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a340c518 | [#36](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/36/) | 2024-02-26 19:22:43 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-192122-a340c5-pr4796.zip) |
| :heavy_check_mark: | a340c518 | [#36](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/36/) | 2024-02-26 19:23:01 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-192118-a340c5-pr4796.aar) |
| :heavy_check_mark: | a340c518 | [#36](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/36/) | 2024-02-26 19:24:24 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-192118-a340c5-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | ff1d4aed | [#37](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/37/) | 2024-02-26 19:37:37 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-193613-ff1d4a-pr4796.zip) |
| :heavy_check_mark: | ff1d4aed | [#37](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/37/) | 2024-02-26 19:37:38 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-193613-ff1d4a-pr4796.aar) |
| :heavy_check_mark: | ff1d4aed | [#37](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/37/) | 2024-02-26 19:39:12 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-193613-ff1d4a-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | 5fc64d34 | [#38](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/38/) | 2024-02-26 19:54:50 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-195308-5fc64d-pr4796.zip) |
| :heavy_check_mark: | 5fc64d34 | [#38](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/38/) | 2024-02-26 19:55:38 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-195308-5fc64d-pr4796.aar) |
| :heavy_check_mark: | 5fc64d34 | [#38](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/38/) | 2024-02-26 19:56:23 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-195308-5fc64d-pr4796.zip) |
| | | | | | | |
| :heavy_check_mark: | ef57b8e8 | [#39](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/39/) | 2024-02-26 20:06:13 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-200453-ef57b8-pr4796.zip) |
| :heavy_check_mark: | ef57b8e8 | [#39](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/39/) | 2024-02-26 20:06:41 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-200453-ef57b8-pr4796.aar) |
| :heavy_check_mark: | ef57b8e8 | [#39](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/39/) | 2024-02-26 20:07:52 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-200453-ef57b8-pr4796.zip) |
| :heavy_multiplication_x: | ef57b8e8 | [#39](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/39/) | 2024-02-26 20:34:46 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/39//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 913e95be | [#40](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/40/) | 2024-02-26 20:44:59 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-204338-913e95-pr4796.zip) |
| :heavy_check_mark: | 913e95be | [#40](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/40/) | 2024-02-26 20:45:21 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-204338-913e95-pr4796.aar) |
| :heavy_check_mark: | 913e95be | [#40](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/40/) | 2024-02-26 20:46:36 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-204338-913e95-pr4796.zip) |
| :heavy_multiplication_x: | 913e95be | [#40](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/40/) | 2024-02-26 21:17:16 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/40//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 62eec2de | [#41](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/41/) | 2024-02-26 21:21:38 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240226-211954-62eec2-pr4796.zip) |
| :heavy_check_mark: | 62eec2de | [#41](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/41/) | 2024-02-26 21:21:39 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240226-211954-62eec2-pr4796.aar) |
| :heavy_check_mark: | 62eec2de | [#41](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/41/) | 2024-02-26 21:22:43 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240226-211954-62eec2-pr4796.zip) |
| :heavy_multiplication_x: | 62eec2de | [#41](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/41/) | 2024-02-26 21:55:04 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/41//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 30c6c58c | [#42](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/42/) | 2024-02-27 08:05:41 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240227-080424-30c6c5-pr4796.zip) |
| :heavy_check_mark: | 30c6c58c | [#42](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/42/) | 2024-02-27 08:06:13 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240227-080420-30c6c5-pr4796.aar) |
| :heavy_check_mark: | 30c6c58c | [#42](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/42/) | 2024-02-27 08:07:58 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240227-080424-30c6c5-pr4796.zip) |
| :heavy_multiplication_x: | 30c6c58c | [#42](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/42/) | 2024-02-27 08:38:32 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/42//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a0cb896b | [#43](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4796/43/) | 2024-02-27 09:24:16 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240227-092300-a0cb89-pr4796.zip) |
| :heavy_check_mark: | a0cb896b | [#43](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4796/43/) | 2024-02-27 09:24:39 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240227-092256-a0cb89-pr4796.aar) |
| :heavy_check_mark: | a0cb896b | [#43](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4796/43/) | 2024-02-27 09:26:24 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240227-092256-a0cb89-pr4796.zip) |
| :heavy_check_mark: | a0cb896b | [#43](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/43/) | 2024-02-27 09:58:36 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4796/43//consoleText) |

osmaczko committed:
chore: unskip flaky tests

osmaczko committed:
fix: prevent `panic: send on closed channel`

osmaczko committed:
fix: prevent messenger being started twice

Previously, Messenger was `Start`ed multiple times, which resulted in
the shutdown process not being invoked on previously initialized
Messenger's sub-instances. This led to the failure of MVDS instance
shutdown causing massive error logs due to the attempts to read from a
closed database.

osmaczko committed:
chore: introduce silent test logger config

osmaczko committed:
chore: add capability to rerun failed tests

- Use `gotestsum` to generate junit test reports and re-run failed tests.
- Archive re-run reports in `${package_dir}/report_rerun_fails.txt`.

jrainville reviewed (approved):
This looks awesome!

osmaczko committed:
Revert "Comment out all logged flaky tests"

This reverts commit 0bd4a06edc0893752378c1a57b35364144e3e169.

osmaczko committed:
chore: skip failing tests
